src/js/editor/event-manager.js

```diff
@@ -45,6 +45,14 @@ export default class EventManager {
     this._textInputHandler.register(inputHandler);
   }
 
+  unregisterInputHandler(name) {
+    this._textInputHandler.unregister(name);
+  }
+
+  unregisterAllTextInputHandlers() {
+    this._textInputHandler.destroy();
```

Comment:
Ah, yes, you're right. Although in the future I could see the implementation of `destroy` changing in a way that makes it not possible to continue to register new handlers, so re-creating it feels good. thanks!